lib/fontcustom/generator/template.rb

```diff
@@ -201,6 +201,16 @@ def glyph_properties
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased;
   font-smoothing: antialiased;|
+        if @options[:ligature]
+          properties += %Q|
+  -webkit-text-rendering: optimizeLegibility;
+  -moz-text-rendering: optimizeLegibility;
+  -ms-text-rendering: optimizeLegibility;
+  -o-text-rendering: optimizeLegibility;
+  text-rendering: optimizeLegibility;|
```

Comment:
This rule has a significant performance cost. Will this feature work without this rule?